### DISCUSSION TITLE
[Announcement] Homework 1 is out

### DISCUSSION THREAD

**Discussion by goodtaeeun** [prosyslab-classroom/cs424-program-reasoning#299] [Announcements]
Hi all, this is an announcement about homework 1.

Homework 1 is a programming assignment requiring you to implement the SmaLLVM interpreter.

Here is the GitHub classroom link for this homework:
https://classroom.github.com/a/kf8DV030

Be sure to **PUSH** the final commit and submit it to **[Gradescope](https://www.gradescope.com/courses/553973)** in order to get graded.

Due is **9.14(Thurs) 23:59:59**, and late submission is available until **9.16(Sat) 23:59:59**.

As explained in the README file, we will check the coverage of your code using the basic tests + your custom tests.
Refer to this [document](https://docs.google.com/presentation/d/1rS39M1dui3khqTc6joWniNrxH3x87nGgOrWHiMAL4DU/edit?usp=sharing) for how to add your custom tests.

Hope you have fun,
Thanks.

**Comment by cwlo2F** [comment]
It appears that the due of homework 1 is incorrectly set on Gradescope. Would you please clarify this?

**Comment by goodtaeeun** [comment]
Thank you for pointing out the discrepancy. I have updated the due of Gradescope to be consistent with the announcement.
